packages/ui/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "ui",
```

Comment:
Just a minor comment, but I feel like a more unique name would make this code more accessible.
For example, when I see `import { Button } from 'ui';`, my first impression is that `ui` must be a local "top level" folder.  If it was named something more unique, like `shared-ui` or something, it would be easier to remember that this is coming from a different package.